src/menu-data.js

```diff
@@ -0,0 +1,95 @@
+export const navMenuDataMain = [
+  {
+    text: 'About',
+    url: 'about',
```

Comment:
I don't think yuo can actually select this and there is nothing on that page

Reply:
this is precisely the point. One remaining problem is that these items should become selectable. Orna has some content for these pages prepared. I tried a bit to make it work with vaadin-menus but so far no luck...